README.md

```diff
@@ -1 +1,48 @@
-# todo-list-v1
\ No newline at end of file
+# To-do list
+
+> This is a project that displays a list of things that one wants to get done or that need to get done.
+# Screenshot
+![screenshot](screenshot.png)
+# Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## Live Demo
+
+[Live Demo Link](https://toliboff.github.io/todo-list-v1/)
+
+# Getting started
+
+## To get a local copy of this repository kindly follow the steps below.
+- Scroll to top of this current repository
+- Click on the `Code` button with background color green on the right end corner
+- Click on the clipboard icon on the extreme right of the dropdown to copy the repository link
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Type `git clone (copied link)` on the currently opened terminal or command prompt
+- Remember to change `(copied link)` to `https://github.com/toliboff/todo-list-v1.git` which is the name of the repository
```

Comment:
[OPTIONAL]

- I suggest you to add the step on how to run the project in local after cloning.

you can tell the step like 

1. `npm install`
2. `npm run build`
3. `npm start`